.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1738,26 +1373,70 @@ public class FineTuningClient {
         public virtual ClientResult GetJobs(string after, int? limit, RequestOptions options);
         public virtual Task<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
     }
+    public class FineTuningJob {
+        public DateTimeOffset CreatedAt { get; }
+        public FineTuningJobError Error { get; }
+        public DateTimeOffset? EstimatedFinish { get; }
+        public string FineTunedModel { get; }
+        public DateTimeOffset? FinishedAt { get; }
+        public FineTuningJobHyperparameters? Hyperparameters { get; set; }
+        public string Id { get; }
+        public string Model { get; }
+        public string Object { get; }
+        public string OrganizationId { get; }
+        public IReadOnlyList<string> ResultFiles { get; }
```

Comment:
I think we'll want this to be `ResultFileIds`

Reply:
same as before. But if we're using `ids` here then during job creation it ought to also be file id rather than file name